nixos/modules/virtualisation/docker-containers.nix

```diff
@@ -43,6 +63,25 @@ let
         '';
         };
 
+        hostname = mkOption {
+          type = with types; nullOr str;
+          description = "Set the hostname inside the container. Not required if using Host networking.";
+          default = null;
+          example = "MyFancyDockerContainer";
+        };
+
+        labels = mkOption {
+          type = with types; listOf str;
```

Comment:
Labels seem to require unique keys, so an attrset might be worth considering here as well.

Reply:
Resolved in 35b9985